doc-content/drools-docs/src/main/asciidoc/DecisionEngine/performance-tuning-decision-engine-ref.adoc

```diff
@@ -36,3 +36,15 @@ try {
 
 For information about built-in event listeners and debug logging in the {DECISION_ENGINE}, see xref:engine-event-listeners-con_decision-engine[].
 --
+
+Use lambda externalization for executable model::
+Enable lambda externalization which optimizes the memory consumption at runtime. It rewrites lambdas that are generated and used in the executable model, enabling you to reuse the same lambda multiple times with all the patterns and the same constraint. The executable model becomes garbage collectible once the rete and phreak is instantiated.
```

Comment:
Thanks for providing the inputs @csherrar. 
How about changing the second sentence as follows to avoid passive form: 
_It rewrites lambdas that are generated and used in the executable model. This enables you to reuse the same lambda multiple times with all the patterns and the same constraint._

Reply:
Looks good to me, good job, @hmanwani-rh.